servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/WriteSingleSubscriber.java

```diff
@@ -73,6 +73,11 @@ public void channelWritable() {
         // No op.
     }
 
+    @Override
+    public void protocolPayloadComplete() {
+        // No op.
```

Comment:
Made it noop because it doesn't seem relevant to Single and not sure we should complicate the state machine for this.

Reply:
If we are adding a generic method to the `WritableListener` we should implement it in a way that makes sense for the contract. In this case it does seem like we would want to emit an error if we get `protocolPayloadComplete()` without `onSuccess()`, it will make it easier in the future to reason about the code.